source/reporting/index.html.md.erb

```diff
@@ -58,48 +58,155 @@ You can use the GOV.UK Pay API to:
-    "postcode": "NW16XE",
-    "city": "London",
-    "country": "GB"
-  }
-}
-```
+- `events` to [get the payment's events](/reporting/#get-a-payment-s-events)
+- `refunds` to [get all refunds for the payment](/refunding_payments/#get-all-refunds-for-a-single-payment)
+- `cancel` to [cancel the payment](/making_payments/#cancel-a-payment-that-s-in-progress)
+
+#### card_details
+
+If your user has gone far enough in their
+payment journey, the `card_details` section contains the information they entered when they made the payment. All the fields are strings.
+
+`billing_address` will be `null` if you've [disabled collecting your users’ billing addresses](/integrate_with_govuk_pay/#collecting-your-users-39-billing-addresses).
+
+`expiry_date` is in `MM/YY` format, for example `04/24`.
+
+`card_brand` is one of the following:
+
+- `American Express`
+- `Diners Club`
+- `Discover`
+- `Jcb`
+- `Maestro`
+- `Mastercard`
+- `Union Pay`
+- `Visa`
+
+`card_type` is one of the following:
+
+- `credit`
+- `debit`
+- `null` - if we did not recognise which type of card your user made the payment with, or your user used [Google Pay](/digital_wallets/#restrictions).
+
+#### corporate_card_surcharge
+
+`corporate_card_surcharge` shows the surcharge amount if you've [set up adding corporate card fees](/corporate_card_surcharges/#add-corporate-card-fees).
+
+#### created_date
+
+`created_date` is when you created the payment, in [ISO
+8601 format](https://www.gov.uk/government/publications/open-standards-for-government/date-times-and-time-stamps-standard).
+
+#### delayed_capture
+
+`delayed_capture` is `true` if you're [controlling how long it takes GOV.UK Pay to take ('capture') a payment](/delayed_capture/#delay-taking-a-payment).
+
+#### fee
+
+If you use GOV.UK’s Payment Service Provider (PSP), `fee` is the [PSP transaction fee](https://docs.payments.service.gov.uk/reporting/#psp-fees) that we took from the amount your user paid.
+
+#### moto
+
+`moto` is `true` if you [took the payment over the phone](/moto_payments/#take-a-payment-over-the-phone-moto-payments).
+
+#### net_amount
+
+If you use GOV.UK’s Payment Service Provider (PSP), `net_amount` is the amount that will be paid into your account after we take the [PSP transaction fee](https://docs.payments.service.gov.uk/reporting/#psp-fees). 
 
-The `card_type` is `null` if we did not recognise which type of card your user made the payment with.
+#### payment_id
+
+`payment_id` is the unique id that GOV.UK Pay automatically generated for this payment.
+
+#### payment_provider
+
+`payment_provider` is the name of your payment service provider (PSP), or `sandbox` if you're using a test ('sandbox') account.
+
+#### refund_summary
+
+You can use `refund_summary` to [check if you can refund a payment](/refunding_payments/#check-if-you-can-refund-a-payment).
+
+#### return_url 
+
+`return_url` is the URL that your service hosted for your user to return to, after their payment journey on GOV.UK Pay ended.
```

Comment:
Does "that your service hosted" imply that they did it in the past or as a one-off? Not sure what I'd suggest though.

Reply:
Good point - we say 'service hosts' elsewhere so I'll switch to that.